src/pages/category/Category.tsx

```diff
@@ -0,0 +1,23 @@
+import { CategoryCard } from "components";
+import "./category.css";
+const categoryData = [
+  {
```

Comment:
Suggestion: You can do API call here and get categoryData from it.

Reply:
yes I have yet to implement the backend and api calls.